src/presentation/pages/Creatures/Carousel/CreaturesCarousel.tsx

```diff
@@ -31,9 +31,10 @@ export const CreaturesCarousel = () => {
         </div>
 
         <Grid container>
-          {carouselItems.map(item => (
+          {carouselItems.map((item, index) => (
```

Comment:
let's avoid using index as key in arrays. Each carousel item has an id property, use it here please